app/classes/api/observation_api.rb

```diff
@@ -6,27 +6,23 @@ class API
   class ObservationAPI < ModelAPI
     self.model = Observation
 
-    self.high_detail_page_length = 10
-    self.low_detail_page_length  = 100
+    self.high_detail_page_length = 100
+    self.low_detail_page_length  = 1000
     self.put_page_length         = 1000
     self.delete_page_length      = 1000
 
     self.high_detail_includes = [
-      :comments,
-      { images: :license },
-      :location,
-      :name,
-      { namings: [:name, { votes: :user }] },
```

Comment:
This looks like we are no longer providing the vote information in this API call.  That makes me a bit worried that we might break a client somewhere, but I don't know much about how the API is actually used so maybe not.  I'd be more confident if they changes were only additive, but even then some folks code API input stuff very strictly so even additive changes can cause breakage.  The traditional thing to do is to actually version the API, but that's not a simple change.  It's not clear to me why you are changing the results of the API call in this ticket which seems like it's mostly about easy performance wins.  Makes it seem a lot more risky to me.  Or maybe I don't understand what these changes mean and the results aren't actually being changed by this.

Reply:
Yes, the diff is very confused.  I did only add, not subtract.  The one change I made that could possibly break existing API users was that I moved the votes "up".  That is, I include the namings exactly like before, but instead of including the corresponding votes under each naming, I show all of the votes in one array at the same level as the namings.  To my knowledge no one has used any of that information yet.  I know specifically that iNat does not, even they they wanted the information.  I think the votes were too "hidden", that's why they didn't see them.  This change will make them more visible.  (iNat wants to give users the ability to import MO observations using the owner's preferred name instead of the consensus name.)